pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+func updateEndpointAccessCmd(cmd *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	cmd.ClusterConfig = cfg
+
+	cmd.SetDescription("update-cluster-api-access", "Update cluster API endpoint configuration", "")
```

Comment:
Let's keep this consistent with config file, so I think it should be `update-cluster-endpoints`

Reply:
Also, 'Cluster API' means something else, let's avoid using it here;  let's change the description too

```suggestion
	cmd.SetDescription("update-cluster-endpoints", "Update Kubernetes API endpoint configuration", "")
```

Reply:
I like the name change, I'm thinking though that I should change the description slightly then to "Update Kubernetes API endpoint access configuration" to disambiguate it from changing the URL for the API endpoint.  Thoughts?

Reply:
changed the name and the description.